tiktoken/core.py

```diff
@@ -6,15 +6,16 @@
 
 from tiktoken import _tiktoken
 
+from typing import Dict, List, Tuple, Set, FrozenSet
```

Comment:
Please let me know which formatter you are using (black?)
I'll sort the imports and reformat the code as you wish.